init-tools.sh

```diff
@@ -1,25 +1,25 @@
 #!/usr/bin/env bash
 
 __scriptpath=$(cd "$(dirname "$0")"; pwd -P)
-__init_tools_log=$__scriptpath/init-tools.log
-__PACKAGES_DIR=$__scriptpath/packages
-__TOOLRUNTIME_DIR=$__scriptpath/Tools
-__DOTNET_PATH=$__TOOLRUNTIME_DIR/dotnetcli
-__DOTNET_CMD=$__DOTNET_PATH/dotnet
+__init_tools_log="$__scriptpath/init-tools.log"
```

Comment:
@dseefeld do you recall what made you add the quoting? I'm curious as I'm trying to sync with these changes and that shows up in the diff so I'm not sure if it is necessary or not.

Reply:
Probably me. I like to have these in general to keep the window open to supporting spaces in the repo path in the future, although I don't think we do now.

More recently, I've noticed that the quotes don't seem necessary for `bar='hello there'; foo=$bar`, but it *is* needed for "actual" commands to avoid breaking into multiple args. (Can be especially nasty in an `rm -rf $path`.)

Reply:
As part of my merge I left the quotes but I couldn't find them as part of any check-in history in the core repo's which is why I asked. We really need our script update :)